source/install/docker-setting-up.rst

```diff
@@ -105,12 +106,12 @@ This command does the following:
 * |opt.pmm-server.latest| is the name and version tag of the image
   to derive the container from.
 
-.. _pmm.docker.specific-version:
+.. _specific-version:
```

Comment:
Changing labels may be not the best idea, as some other part may use them to generate the link to this place.  So renaming labels is safe only if label is never met in docs outside of this exact place.

Reply:
Only changed the label because the tag to itself didn't work on it's own page...if I click the text "installing and using specific docker version" it doesn't actually link to that tag correctly...it was the only tag that was prefixed by _pmm.docker.<tagname>...above "Creating and Launching the PMM Server Container" does work so I mimicked that tag.  If you let me know which format is correct I'll recommit with the fix.